features/step_definitions/file_steps.coffee

```diff
@@ -26,6 +27,13 @@ module.exports = ->
     addToJsonFile filePath, content, done
 
 
+  @Given /^I have configured "([^"]*)" to be (.+)$/, (key, value, done) ->
```

Comment:
I'd prefer this step to just be `/^I have configured "([^"]*)" to be true$/` for now. Might as well keep it as simple as possible